src/pages/management/MachinePage.tsx

```
@@ -118,27 +148,32 @@ const MachinePage = () => {
-          initial: row.id,
+          initial: item.key === "itemId" ? row.id : row[item.key],
         };
-      }
-      return {
-        ...item,
-        initial: item.key === "itemId" ? row.id : row[item.key],
-      };
-    });
-    setModalInputs(addInitialInputs);
+      });
+      addInitialInputs && setModalInputs(addInitialInputs);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)